sematic/cli/settings.py

```diff
@@ -6,69 +6,77 @@
 
 # Sematic
 from sematic.cli.cli import cli
-from sematic.config.server_settings import (
-    ServerSettingsVar,
-    delete_server_settings,
-    get_active_server_settings,
-    set_server_settings,
-)
-from sematic.config.settings import dump_settings
-from sematic.config.user_settings import (
-    UserSettingsVar,
-    delete_user_settings,
-    get_active_user_settings,
-    set_user_settings,
+from sematic.config.server_settings import ServerSettings
+from sematic.config.settings import (
+    delete_plugin_setting,
+    dump_settings,
+    get_active_settings,
+    import_plugin,
+    set_plugin_setting,
 )
+from sematic.config.user_settings import UserSettings
 
 
 @cli.group("settings")
 def settings() -> None:
     pass
 
 
-@settings.command("show", short_help="Show the currently active user settings")
-def show_user_settings_cli() -> None:
+@settings.command("show", short_help="Show the currently active settings")
+def show_settings_cli() -> None:
     """
     Show the currently active user settings.
     """
-    settings_dump = dump_settings(get_active_user_settings())
+    settings_dump = dump_settings(get_active_settings())
     click.echo(f"Active user settings:\n\n{settings_dump}")
 
 
 @settings.command("set", short_help="Set a user settings value")
 @click.argument("var", type=click.STRING)
 @click.argument("value", type=click.STRING)
-def set_user_settings_cli(var: str, value: str) -> None:
+@click.option("-p, --plugin", "plugin_path", default=UserSettings.get_path())
```

Comment:
The fact that this is used in the `sematic settings` flow suggests that these settings are somehow a part if user settings, even though this is not always necessarily true.

The `-p/--plugin` option suggests an optional piece of information. If I were to remove it, I would expect `sematic settings set VAR value` to work, which isn't true as that setting actually belongs to the plugin.

Also, in order to set settings under a particular plugin, that plugin must already exist configured under a scope. Otherwise, settings could be set on incorrect plugin paths. A command for configuring plugins under scopes is necessary in this design.

This is a major change, and the syntax of the cli is the entry path for anyone to configure the plugins. I would rather invest more in it now that to have complicated deprecations and migration later.

I propose a symmetrical syntax:
```
$ sematic user-settings set VAR value
$ sematic server-settings set VAR value
$ sematic plugin-settings set plugin.path VAR value

$ sematic plugin-settings add-plugin plugin.path scope
$ sematic plugin-settings remove-plugin plugin.path scope
```